src/astx/tools/transpilers/python.py

```diff
@@ -495,6 +495,11 @@ def visit(self, node: astx.WalrusOp) -> str:
         """Handle Walrus operator."""
         return f"({self.visit(node.lhs)} := {self.visit(node.rhs)})"
 
+    @dispatch  # type: ignore[no-redef]
+    def visit(self, node: astx.AugAssign) -> str:
+        """Handle Augmented assign operator."""
+        return f"({self.visit(node.target)} {node.aug_op} {self.visit(node.value)})"
```

Comment:
```suggestion
        return f"{self.visit(node.target)} {node.op_code} {self.visit(node.value)}"
```

It is probably not necessary to use parenthesis here